jib-core/src/main/java/com/google/cloud/tools/jib/cache/Cache.java

```diff
@@ -59,10 +63,14 @@ private static CacheMetadata loadCacheMetadata(Path cacheDirectory)
     }
 
     try {
-      CacheMetadataTemplate cacheMetadataJson =
-          JsonTemplateMapper.readJsonFromFile(cacheMetadataJsonFile, CacheMetadataTemplate.class);
-      return CacheMetadataTranslator.fromTemplate(cacheMetadataJson, cacheDirectory);
-
+      // channel is closed by inputStream.close()
```

Comment:
Maybe we should include these as methods on `JsonTemplateMapper`? Like `JsonTemplateMapper#readJsonFromFile` and `Blobs#writeToFile`?